conf/defaults.ini

```diff
@@ -1194,40 +1194,46 @@ facility =
 tag =
 
 [log.frontend]
-# Should Faro javascript agent be initialized
+# Enables the Grafana Faro javascript agent integration for capturing frontend performance and error events.
 enabled = false
 
-# Custom HTTP endpoint to send events to. Default will log the events to stdout.
-custom_endpoint =
+# Custom HTTP endpoint to send Grafana Faro events to. Default will send events to /log-grafana-javascript-agent and log the events to stdout.
+custom_endpoint = /log-grafana-javascript-agent
```

Comment:
Is this default also set in-code somewhere? Maybe we could remove it?

Reply:
As you've seen, yeah. I've just applied the default here from the code to be consistent with all the others https://github.com/grafana/grafana/blob/c0ca628f4d69771b12b97efb6dbd71fa39b9c87e/pkg/setting/setting_grafana_javascript_agent.go#L27